suite/tests/CMakeLists.txt

```diff
@@ -1403,7 +1403,11 @@ function(torun test key source native standalone_dr dr_ops exe_ops added_out pas
   endif ()
 
   if (DEFINED ${key}_sudo)
-    set(rundr "sudo;${rundr}")
+    if (DEFINED ${key}_nodr)
```

Comment:
I thought _nodr was already setting up the command to run to not run under DR: looking -- it only does it for runcmp or sthg?

Reply:
> I thought _nodr was already setting up the command to run to not run under DR

Yes at L1522.

But note that the change here at L1406 is not to achieve that. It is to run `${exepath}` with sudo. (I found that burst tests did not run with sudo even though I set _sudo)